api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -64,6 +64,16 @@ public FireCloudServiceImpl(Provider<WorkbenchConfig> configProvider,
     this.retryHandler = retryHandler;
   }
 
+  private void checkAndAddRegistered(WorkspaceIngest workspaceIngest) {
+    if (configProvider.get().firecloud.enforceRegistered) {
+      ArrayList<ManagedGroupRef> authDomain = new ArrayList<ManagedGroupRef>();
+      ManagedGroupRef registeredDomain = new ManagedGroupRef();
+      registeredDomain.setMembersGroupName(configProvider.get().firecloud.registeredDomainName);
+      authDomain.add(registeredDomain);
+      workspaceIngest.setAuthorizationDomain(authDomain);
```

Comment:
nit: I would probably remove the `authDomain` variable entirely and inline an `ImmutableList.of(registeredDomain)` here